src/main/templates/facilities.hbs

```diff
@@ -31,10 +29,8 @@
     <tbody>
       {{#data}}
       <tr>
-        <td>
-          <pre>{{code}}</pre>
```

Comment:
`<pre>` was intended to generate Courier text, implying that the string is the exact value to use. What about `<code>{{code}}</code>` instead?